main.lua

```diff
@@ -0,0 +1,63 @@
+Kaizer = {img=nil, x=200, y=0, w=0, h=0, speed=500}
+EnemyCars = {}
+imageKaizer = nil
+imageCar1 = nil
+imageCar2 = nil
+imageCar3 = nil
+
+function love.load()
+  imageKaizer = love.graphics.newImage("Kaizer.JPG")
+  imageCar1 = love.graphics.newImage("Car1.JPG")
+  imageCar2 = love.graphics.newImage("Car2.JPG")
+  imageCar3 = love.graphics.newImage("Car3.JPG")
+
+  Kaizer = {img=imageKaizer, x=200, y=0, w=imageKaizer:getWidth(), h=imageKaizer:getHeight()}
+  C1 =  {img=imageCar1, x=0, y=100, w=imageCar1:getWidth(), h=imageCar1:getHeight()}
+  table.insert(EnemyCars, C1)
+  C2 = {img=imageCar2, x=0, y=200, w=imageCar2:getWidth(), h=imageCar2:getHeight()}
+  table.insert(EnemyCars, C2)
+  C3 = {img=imageCar3, x=0, y=300, w=imageCar3:getWidth(), h=imageCar3:getHeight()}
+  table.insert(EnemyCars, C3)
+end
+
+function love.update(dt)
+  for i, car in ipairs(EnemyCars) do

```

Comment:
I'm a bit confused with this loop, why are the keyboard stuff in this loop? Or am I missing something..